docs/_documentations/mdt-eclipse-managingprojects.md

```diff
@@ -18,7 +18,7 @@ You can work with your Codewind projects from the **Codewind Explorer** view in
 - Start typing **Codewind** in the filter field or locate and expand the **Codewind** entry in the list.
 - Select **Codewind Explorer** and click **Open**.
 
-To create a new project or import an existing one, use the context menu on the **Local Projects** item in the **Codewind Explorer** view.  Once you have a project, the first thing you might want to do is import your project into the Eclipse workspace so you can start editing files. This also makes your source available for debugging.
+To create a new project or import an existing one, use the context menu on the **Local** item in the **Codewind Explorer** view. After you have a project, the first thing you might want to do is import your project into the Eclipse workspace so you can start editing files. This also makes your source available for debugging.
```

Comment:
I may have missed this in the readme but the projects are now imported automatically for the user so delete: "After you have a project, the first thing you might want to do is import your project into the Eclipse workspace so you can start editing files. This also makes your source available for debugging."

Reply:
I checked the readme and it is ok.